CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
I don't want to reduce the minimum cmake version, because it doesn't support gtest_discover_tests. 
Can you not upgrade cmake on circle-ci?

Reply:
This has come up before: https://github.com/ebruck/radiotray-ng/pull/91

Reply:
I revert this line, and install cmake 3.14.3 in ubuntu 16.04 docker image (see  `docker/ubuntu/16.04/Dockerfile`). So, now this fixed.